docs/extensions/server/configuration.rst

```diff
@@ -18,7 +18,52 @@ Inject Session Conf with Custom Config Advisor
 
 .. versionadded:: 1.5.0
 
-.. caution:: unstable
+The step of inject session conf
```

Comment:
Can we also elaborate a bit about what is it and what are the common use cases of it?